### PR TITLE
refactor: remove blockchain app example

### PR DESCRIPTION
**Pull request by MatejVukosav**
Example is in https://github.com/calimero-network/demo-blockchain-integrations

### CODE REVIEW

**Comment by MatejVukosav** [comment]
comment-test-2

**Comment by MatejVukosav** [comment]
comment-test-draft

**Review by miraclx — COMMENTED** [review]
comment test

File: Cargo.toml [hunk 1]
```diff
@@ -33,7 +33,6 @@ members = [
     "./apps/only-peers",
     "./apps/gen-ext",
     "./apps/visited",
-    "./apps/blockchain",
```
Comment (MatejVukosav) [inline Cargo.toml:36]: review-test-comment

**Review by alenmestrov — APPROVED** [review]

**Review by miraclx — APPROVED** [review]

**Pull request merged**